stock_reserve/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "license": "AGPL-3",
     "complexity": "normal",
     "website": "https://github.com/OCA/stock-logistics-warehouse",
-    "depends": ["stock"],
+    "depends": ["stock", "sale_stock"],
```

Comment:
@aliciagaarzo You cannot add a dependency just to fix tests :sweat_smile:

Reply:
The tests fail me because I don't have the dependency and I added it but I remove it